### ISSUE TITLE
Voting - Cosmos Gov V1

### ISSUE THREAD

**Issue by mhagel** [hicommonwealth/commonwealth#3361]
Follow up to #3159

AC: I can vote on a kyve proposal

Background: if no active kyve proposals to test with, we may need to use a local kyve devnet node.

**Comment by mhagel** [comment]
Submitting onchain proposal and Voting works without updating the TX signing client code, but there is a misleading error message. 

https://www.loom.com/share/5fb8ee2d747d4cccb959ef0926d5788a

This apparently happens to all transactions in cosmjs against a post 0.46 SDK chain. I tried their (very recent) [patch](https://github.com/cosmos/cosmjs/issues/1353), but it didn't resolve this. I also tried using telescope's signing client - same result.

It is related to the tendermint client. When I tried to use a more updated one, it broke more than it fixed.

At this point I think the best idea is to handle this error more gracefully.
